<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Branch</class>
 <widget class="QDialog" name="Branch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GitUI Branch</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="3" rowspan="5">
    <widget class="QTextBrowser" name="wyswietlacz"/>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="5" column="0" rowspan="3" colspan="3">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="labelError">
       <property name="text">
        <string>labelError</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButtonAddBranch">
       <property name="text">
        <string>Add branch</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelInfo">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>labelInfo</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLineEdit" name="lineEditBranchName"/>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButtonDeleteBranch">
       <property name="text">
        <string>Delete selected branch:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButtonMerge">
       <property name="text">
        <string>Merge</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBox_2"/>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButtonCheckout">
       <property name="text">
        <string>Checkout</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="pushButtonBack">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Your branches in </string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelBranch">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Currently on branch:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QRadioButton" name="radioButton">
     <property name="text">
      <string>Modify branches</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Nimbus Roman No9 L</family>
       <pointsize>16</pointsize>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string>Change branch</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
